test/integration/rpc-server-controller.spec.ts

```diff
@@ -99,4 +100,103 @@ test('RPC Server Controller', function ({ components, stubComponents }) {
       await removeFriendship(db, id, rpcClient.authAddress)
     })
   })
+
+  describe('getPendingFriendshipRequests', function() {
+    it('returns pending friendship requests successfully', async () => {
```

Comment:
If we want to add more robustness to the tests, we could add a test that exercises the pagination, what do you think about adding one?